lib/cocoapods-binary-cache/pod-rome/xcodebuild_command.rb

```diff
@@ -81,16 +93,56 @@ def build_for_sdk(sdk)
       )
     end
 
+    def archive_for_sdk(sdk, target)
+      PodPrebuild::XcodebuildCommand.xcodebuild_archive(
+        sandbox: sandbox,
+        target: target,
+        configuration: configuration,
+        sdk: sdk,
+        build_dir: build_dir,
+        args: sdk == simulator ? @build_args[:simulator] : @build_args[:device]
+      )
+    end
+
     def create_xcframework(target)
-      non_framework_paths = Dir[target_products_dir_of(target, sdks[0]) + "/*"] - [framework_path_of(target, sdks[0])]
+      non_framework_paths = Dir[target_products_dir_of(target, sdks[0]) + "/*"] \
+        - [ framework_path_of(target, sdks[0]),         # prevent collect .framework
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/grab/cocoapods-binary-cache/blob/b144f30fb41bd34e49fceb072710e5f2c3a3e710/lib/cocoapods-binary-cache/pod-rome/xcodebuild_command.rb#L109"></span>Layout/SpaceInsideArrayLiteralBrackets: Do not use space inside array brackets.</td>
    </tr>
  </tbody>
</table>